src/Backend/Modules/Blog/DataFixtures/LoadBlogPosts.php

```diff
@@ -60,4 +65,26 @@ public function load(SpoonDatabase $database): void
             ]
         );
     }
+
+    private function insertCategory(): Category
+    {
+        $meta = new Meta(
+            'BlogCategory for tests', false,
```

Comment:
same

Reply:
fixed in 4c322d4485af9cf7eb3496e8b351045e61c6ff60